pkgs/by-name/al/alvr/package.nix

```
@@ -130,7 +125,7 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     install -Dm755 ${src}/alvr/xtask/resources/alvr.desktop $out/share/applications/alvr.desktop
-    install -Dm644 ${src}/resources/alvr.png $out/share/icons/hicolor/256x256/apps/alvr.png
+    install -Dm644 ${src}/resources/ALVR-Icon.svg $out/share/icons/hicolor/256x256/apps/ALVR-Icon.svg
```

The desktop file is still looking for an icon named alvr, and vector icons should be added to the `scalable` folder and not to a specific resolution.

```suggestion
    install -Dm644 ${src}/resources/ALVR-Icon.svg $out/share/icons/hicolor/scalable/apps/alvr.svg
```

---

thanks for spotting!